v2/spanner-common/pom.xml

```diff
@@ -52,6 +52,73 @@
                         <phase>none</phase>
                     </execution>
                 </executions>
+            </plugin>
+		    <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>${jacoco.version}</version>
+                <configuration>
+                    <excludes>
+                        <!-- Excluding exception classes -->
+                        <exclude>**/*Exception.*</exclude>
+                        <!-- Excluding auto-generated classes. -->
+                        <exclude>**/*AutoValue_*</exclude>
+                    </excludes>
+                    <rules>
+                        <rule>
+                            <element>PACKAGE</element>
+                            <includes>
+                                <include>com.google.cloud.teleport.spanner</include>
+                                <include>com.google.cloud.teleport.spanner.**</include>
+                            </includes>
+                            <excludes>
+                                <!-- Excluding classes covered by integration tests. -->
+                                <exclude>com.google.cloud.teleport.spanner.ddl.InformationSchemaScanner</exclude>
+                                <!-- Excluding generated proto java files -->
+                                <exclude>com.google.cloud.teleport.spanner.proto</exclude>
+                            </excludes>
+                        </rule>
+                    </rules>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>default-prepare-agent</id>
+                        <goals>
+                            <goal>prepare-agent</goal>
+                        </goals>
+                    </execution>
+                    <execution>
+                        <id>default-report</id>
+                        <phase>prepare-package</phase>
+                        <goals>
+                            <goal>report</goal>
+                        </goals>
+                    </execution>
+                    <execution>
```

Comment:
Also, Line 97 to 120 is not needed. This block is for coverage check and The coverage check is now handled through Codecov. Not needed here.